tests/e2e/conftest.py

```diff
@@ -1,4 +1,4 @@
 def pytest_addoption(parser):
     parser.addoption("--core_url", action="store", default="localhost:6565")
-    parser.addoption("--serving_url", action="store", default="localhost:6565")
-    parser.addoption("--allow_dirty", action="store", default="false")
+    parser.addoption("--serving_url", action="store", default="localhost:6566")
+    parser.addoption("--allow_dirty", action="store", default="true")
```

Comment:
Are you sure about making the default `true` here? I made it false so that when people run e2e tests locally that it wouldnt destroy a real cluster.

Reply:
That sounds prudent 😇

Reply:
Yeah sounds good. I think i'll change the test script to specify allow_dirty options explicitly rather than changing the default